exporter/metadata.py

```diff
@@ -18,12 +53,16 @@ class MetadataException(Exception):
 
 class MetadataProvenance:
     def __init__(self, document_id: str, submission_date: str, update_date: str,
-                 schema_major_version: int, schema_minor_version: int):
+                 schema_major_version: int = None, schema_minor_version: int = None):
         self.document_id = document_id
         self.submission_date = submission_date
         self.update_date = update_date
-        self.schema_major_version = schema_major_version
-        self.schema_minor_version = schema_minor_version
+
+        if schema_major_version:
```

Comment:
What happens if the `schema_major_version` is None?
Also this field is never used inside this class: `self.schema_major_version`. Is that intentional? The same for the `self.schema_minor_version`
Tx